packages/dropdowns.next/src/index.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+export { Combobox } from './elements/combobox/Combobox';
+export { Field } from './elements/combobox/Field';
+export { Hint } from './elements/combobox/Hint';
+export { Label } from './elements/combobox/Label';
+export { Message } from './elements/combobox/Message';
+export { Option } from './elements/combobox/Option';
+export { OptGroup } from './elements/combobox/OptGroup';
+export { Tag } from './elements/combobox/Tag';
```

Comment:
How to use the `Tag` component is not very clear, how is this used within the context of `Combobox`?

Reply:
It isn't ever used directly. We export here in order to generate the downstream prop table so that consumers have a reference for `tagProps` on the `Option` API. We'll make this clear on the website.